test/test_models.py

```diff
@@ -58,6 +59,36 @@ def test_call(override, base_url, swagger_20):
     assert not records
 
 
+def test_case_deepcopy(swagger_20, openapi_30):
```

Comment:
`openapi_30` is not used. I assume that the idea is to test this behavior under both spec versions. If yes, then I propose to use a parametrized fixture:

```
@pytest.fixture(params=["swagger_20", "openapi_30"]
def schema(request):
    return request.getfixturevalue(request.param)
```

and then use `schema` fixture in the test

Reply:
I needed a schema to initialize the `Endpoint`. I initially planned to use both, but changed my plan and forgot to remove `openapi_30`.